CMakeLists.txt

```diff
@@ -348,7 +348,13 @@ message(STATUS "Using Python: ${CHPL_CMAKE_PYTHON}")
 add_subdirectory(compiler)
 add_subdirectory(frontend)
 add_subdirectory(tools/chpldoc)
-add_subdirectory(tools/chpldef)
+
+# We are not ready to include 'chpldef' in an official release just yet.
+if (NOT CMAKE_BUILD_TYPE STREQUAL "Release")
+  if (EXISTS tools/chpldef)
```

Comment:
Do we need to check this? We weren't before and I think this will always be true?

Reply:
The script that generates the tarball copies files from the source tree into a working space before building them. The source for the new tool isn't copied over, so this line in the `CMakeLists.txt` will break (that's what's causing the nightly test failures). Because this is a new tool that isn't ready yet, we don't want to include it in our releases yet.

Reply:
:+1: